plugins/puma/puma_

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Copyright (c) 2016 Alexey Anikanov (alexey.anikanov@gmail.com)
+# License GPLv2
+# This plugin monitors number of workers, total memory used and average memory per process for puma,
+# a ruby web server built for concurrency http://puma.io
+# Here are the symlinks to enable it
+#
+#        ln -s /usr/share/munin/plugins/puma_ /etc/munin/plugins/puma_average
+#        ln -s /usr/share/munin/plugins/puma_ /etc/munin/plugins/puma_memory
+#        ln -s /usr/share/munin/plugins/puma_ /etc/munin/plugins/puma_processes
```

Comment:
If it makes sense to always have all the graphics, it is usually much better to have a multigraph plugin, as it's called only once.